yarn.lock

```diff
@@ -9194,6 +9194,40 @@ amdefine@>=0.0.4:
   resolved "https://registry.npmjs.org/amdefine/-/amdefine-1.0.1.tgz#4a5282ac164729e93619bcfd3ad151f817ce91f5"
   integrity sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=
 
+amplify-cli-core@2.6.0:
```

Comment:
These yarn.lock changes are unrelated to your PR, can we exclude them? https://github.com/aws-amplify/amplify-cli/pull/10392 will fix the lock file